src/main/java/org/openrewrite/java/migrate/guava/NoGuavaListsNewArrayList.java

```diff
@@ -87,6 +89,11 @@ public J visitMethodInvocation(J.MethodInvocation method, ExecutionContext ctx)
                     maybeAddImport("java.util.ArrayList");
                     return newArrayListCapacity.apply(getCursor(), method.getCoordinates().replace(),
                             method.getArguments().get(0));
+                } else if (NEW_ARRAY_LIST_EXPECTED.matches(method)) {
+                    maybeRemoveImport("com.google.common.collect.Lists");
+                    maybeAddImport("java.util.ArrayList");
+                    return newArrayListCapacity.apply(getCursor(), method.getCoordinates().replace(),
+                            method.getArguments().get(0));
```

Comment:
Ah I'd left this out previously, as there's some nuance here that would be lost, since capacity != size. Best to tackle this separately.